src/content/docs/full-stack-observability/index.mdx

```diff
@@ -3,56 +3,79 @@ title: Full-Stack Observability
 type: landingPage
 tags:
   - Full-Stack Observability
+redirects:
```

Comment:
[nit] Since there are no redirects here, you can omit this.

Reply:
Good catch, just removed it!